_data/accessibility-tests/footer.yml

```
@@ -0,0 +1,100 @@
+title: Footer
+component_status: pass
+test_items:
+# General tests
+  - summary: In the big footer variant, the email input field announces its purpose.
+    summary_additional: When you navigate to the email address input field in the big footer variant, you hear that the input expects the user's email address. Additionally, the input field has the `autocomplete="email"` attribute.
+    test_status: exception
+    test_type: general
+    status_details: The input is appropriately marked as an "email" input type, but does not have the autocomplete attribute in the default component example. Be sure to add the `autocomplete="email"` attribute to the email input element in your implementation of the component.
+    github_issue_number: 6001
+    github_issue_repo: uswds
+    version_tested: 3.8.1
+    wcag_criterion: 1.3.5
+  - summary: Footer font meets color contrast requirements.
+    summary_additional: When you use a color contrast checker to view the footer, contrast between the font color and background color is at least 4.5:1.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.3
+  - summary: Footer elements are visible and functional when screen is magnified.
+    summary_additional: When you zoom in to 400%, footer links and elements are stacked and visible. No left-to-right scrolling is required.
+    test_status: pass
+    test_type: general
```

Just realized this should be marked as a test for zoom, not general.
```suggestion
    test_type: zoom
```

---

Resolved in [436b684](https://github.com/uswds/uswds-site/pull/2760/commits/436b684383d643148985ba2961a81d6d902488e5)